### DISCUSSION TITLE
.block spammers

### DISCUSSION THREAD

**Discussion by Josverl** [micropython/micropython#14688] [General]
@jimmo @dgeorge, could you please block the spammers?
I've already reported them, but cannot directly block them here

**Comment by robert-hh** [comment]
I kept reporting them as well, amd they get blocked, but the spammers change their identity.

**Comment by dhalbert** [comment]
There is a way to prevent interactions by users whose accounts were created recently.
https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-organization

![image](https://github.com/micropython/micropython/assets/2847802/2da2b1d0-df0b-4239-99eb-0f3727b25adc)

**Reply by robert-hh** [reply]
Thanks. I tried that.
Edit: Looks like I cannot do it.

**Reply by dhalbert** [reply]
A recent thread about this kind of spam: https://github.com/orgs/community/discussions/121439.

I think I see a [comment](https://github.com/orgs/community/discussions/121439#discussioncomment-9282974) that notes that the initial posts are not being blocked by the interaction limits above :confused: (edit: but it's confusing)

**Reply by dhalbert** [reply]
> Edit: Looks like I cannot do it.

Maybe an admin can. I mentioned this to @Jimmo in the discord several months ago. He may have turned it on, but it expires after the specified interval, and maybe that expiration happened recently.

**Reply by robert-hh** [reply]
Too bad.

**Reply by dpgeorge** [reply]
Indeed, this setting is just a temporary one.  I'm pretty sure I've enabled it before, but it eventually turns itself off.

I've now enabled it again.

**Reply by Josverl** [reply]
Thanks for actioning, while not airtight, blocking a Spamers user will at least hinder them.
